less/project/columns.less

```diff
@@ -1,3 +1,8 @@
+// Custom classes
```

Comment:
Cheeky of me, can we change to `Custom column classes`

Reply:
Yep, that's more descriptive. Done.

I've also updated the push and pull comments (e.g. "Custom column pull classes" instead of "Custom pull classes"), but happy to change them to whatever.